docs/databases/uploads.md

```diff
@@ -4,19 +4,23 @@ title: Uploading data
 
 # Uploading data
 
-Once you've [enabled uploads](#enabling-uploads), you can [upload files](../exploration-and-organization/collections.md#uploading-data) in CSV format to a collection in Metabase.
+Once you've [enabled uploads](#setting-up-uploads), you can [upload files](../exploration-and-organization/uploads.md) in CSV format to a collection in Metabase.
 
 ![Upload CSV data to a collection in Metabase](./images/upload-to-collection.png)
 
 Uploading CSV data is best suited for ad hoc analysis of spreadsheet data. If you have a lot of data, or will need to update or add to that data regularly, we recommend setting up a way to load that data into a database directly, then connecting Metabase to that database.
 
+## Managing upload settings
+
+To manage upload settings, admins can hit cmd/ctrl + K and search for "Settings - Uploads", or click on the **gear** icon in the upper right and click through **Admin settings** > **Settings** > **Uploads**.
+
 ## Databases that support uploads
```

Comment:
We now have clickhouse too. But it is a bit weird, as Clickhouse is not an official driver (it is a partner driver)